_posts/2020-10-14-distributed-and-failover-realms.adoc

```diff
@@ -2,7 +2,7 @@
 layout: post
 title: 'An overview of new security realm implementations'
 date: 2020-10-14
-tags: elytron wildfly21 securityrealm realm identity distributed database
+tags: wildfly21 securityrealm realm identity distributed database
```

Comment:
Can drop wildfly version, realm, identity, and datatbase

security-realm
distributed
failover